src/views/virtualmachines/details/tabs/disk/tables/disk/DiskRow.tsx

```diff
@@ -23,18 +23,18 @@ const DiskRow: React.FC<
   return (
     <>
       <TableData id="name" activeColumnIDs={activeColumnIDs}>
-        <Split hasGutter>
+        <Stack>
           <SplitItem>
```

Comment:
if using the `Stack` component, you should replace also the `SplitItem` component with `StackItem` for consistency